Sources/Clappr/Classes/Base/Core.swift

```diff
@@ -109,10 +109,11 @@ open class Core: UIObject, UIGestureRecognizerDelegate {
     }
 
     open override func render() {
-        parentView?.addSubviewMatchingConstraints(overlayView)
+        view.addSubviewMatchingConstraints(overlayView)
         containers.forEach(renderContainer)
         addToContainer()
-        parentView?.bringSubviewToFront(overlayView)
+        view.bringSubviewToFront(overlayView)
+        overlayView.clipsToBounds = true
```

Comment:
Just a question, why do we need to set this property?

Reply:
Clips to bounds is what makes the drawer disappear when it goes of of the screen. Clips do bounds doesn’t let subviews to be presented outside the parent view

Reply:
Clips to bounds is what makes the drawer disappear when it goes of of the screen. Clips do bounds doesn’t let subviews to be presented outside the parent view

Reply:
=]